init.yaml

```diff
@@ -1,37 +1,59 @@
+features:
+  - "basic-auth"
+  - "scm-github"
+  - "jwt"
```

Comment:
`jwt-private-key` and `jwt-public-key` are created only when `auth` is true, so you don't need `jwt` as a feature